string-manipulation-1/src/is-lower-cased.js

```diff
@@ -0,0 +1,9 @@
+/* exported isLowerCased */
+const isLowerCased = (word) => {
+  for (const x of word) {
+    if (x.charCodeAt() >= 65 && x.charCodeAt() <= 90) {
+      return false;
+    }
+  }
+  return true;
+};
```

Comment:
We can simplify this! We don't even need a loop! We can convert the whole string to lowercase and compare it to the original string.
```
function isLowerCased(word) {
  return word === word.toLowerCase();
}
```